web/themes/custom/pineapple/sass/views/_view--articles.scss

```diff
@@ -0,0 +1,16 @@
+.article-teaser-container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+
+  @include breakpoint($palm) {
+    flex-direction: row;
+  }
+
+  .views-row {
+    display: block;
+    margin-bottom: $unit * 3;
+    position: relative;
+    width: 100%;
+  }
+}
```

Comment:
This file folder and should inside the layouts folder

Reply:
done